### ISSUE TITLE
Model improvements

### ISSUE THREAD

**Issue by lewfish** [azavea/keras-semantic-segmentation#11]
Right now we get 85.8% accuracy on the Potsdam dataset using a single U-Net-like model trained from scratch on RGBIRD. From reading papers, it seems that this is about the best you can do with a single off-the-shelf model trained from scratch. To improve accuracy, we might explore the following ideas:
* Do some hyperparameter tuning once we have the ability to run lots of experiments in parallel.
* Implement a more proper version of U-Net. The version we have takes some shortcuts for ease of implementation that might result in lower accuracy.
* Use a more state-of-the-art model like 100 Layer Tiramisu, which is like a U-Net but uses DenseNets as its base network. https://arxiv.org/abs/1611.09326
* Train a bunch of models and combine them as an ensemble. Top entries in the contest do this.
* Use a pre-trained model on the RGB channels and then fuse with a trained-from-scratch model on the IR and D channels. How and where you do the fusing probably matters. Top entries in the contest do this.
